assets/src/amp-story-editor-blocks.js

```diff
@@ -52,9 +63,69 @@ domReady( () => {
 			maybeEnqueueFontStyle( block.attributes.ampFontFamily );
 		}
 	}
+
+	renderStoryComponents();
+
+	// Prevent WritingFlow component from focusing on last text field when clicking below the carousel.
+	document.getElementsByClassName( 'editor-writing-flow__click-redirect' )[ 0 ].remove();
```

Comment:
Maybe a bit better to use `querySelector` since we're only needing the first:

```suggestion
	document.querySelector( '.editor-writing-flow__click-redirect' ).remove();
```